core/recovery/recover.go

```diff
@@ -21,7 +21,7 @@ func WrapRecover(lggr logger.Logger, fn func()) {
 			sentry.CurrentHub().Recover(err)
 			sentry.Flush(logger.SentryFlushDeadline)
 
-			lggr.Errorw("goroutine panicked", "panic", err, "stacktrace", string(debug.Stack()))
+			lggr.CriticalW("Recovered goroutine panic", "panic", err, "stacktrace", string(debug.Stack()))
```

Comment:
Bumped these to critical level, and also noticed that this will double report via sentry above and then within the logger too. It could probably be worked around by doing something like (1) changing Critical* signatures to accept a proper `error` and (2) wrapping recovered panics in a `type Recovered` that the logger can check for and then use sentry `Recover()` instead.

Reply:
(Or perhaps just another set of three `Logger.Recovered*` methods would be cleaner :shrug: )

Reply:
Follow up story: https://app.shortcut.com/chainlinklabs/story/22268/sentry-wraprecover-double-reports